<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCustomPlot" name="customplot_raw" native="true">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>411</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customplot_2" native="true">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>230</y>
      <width>401</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_filter">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="sample_rate">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>360</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Read">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>read</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Browse">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>410</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>browse</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customplot_changing" native="true">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>430</y>
      <width>431</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
